src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,96 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_CAPACITY = 16;
+    static final double DEFAULT_LOAD_FACTOR = 0.75;
```

Comment:
```suggestion
public class MyHashMap<K, V> implements MyMap<K, V> {
    static final int DEFAULT_CAPACITY = 16;
    static final double DEFAULT_LOAD_FACTOR = 0.75;
    static final int CAPACITY_MODIFIER = 2;
```
I would suggest having an additional constant for a multiplier